bundles/Billing/Service/BillingService.php

```diff
@@ -802,15 +802,7 @@ public function getProjectWorklogsWithMetadata($projectId)
         $versions = $project->versions;
         $epics = $this->getProjectEpics($projectId);
 
-        // Get custom fields.
-        $customFields = $customFields = $this->get('/rest/api/2/field');
-
-        // Get Epic name field id.
-        $customFieldEpicName = $customFieldEpicLink = array_search(
-            'Epic Name',
-            array_column($customFields, 'name')
-        );
-        $epicNameCustomFieldId = $customFields[$customFieldEpicName]->{'id'};
+        $epicNameCustomFieldId = $this->getCustomFieldId('Epic Name');
```

Comment:
Perhaps this field id should be defined in some configuration rather than searching for it. Do we know that it's always called `Epic Name`? And that it always exists?